src/components/WebexAvatar/WebexAvatar.stories.js

```diff
@@ -1,6 +1,78 @@
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
+import PeopleJSONAdapter from '../../adapters/PeopleJSONAdapter';
+import {PersonStatus} from '../../adapters/PeopleAdapter';
+import person from '../../data/people';
+
 import WebexAvatar from './WebexAvatar';
 
-storiesOf('Webex Avatar', module).add('Avatar with title', () => <WebexAvatar title="Tom Smith" />);
+// Setup for the stories
+const [personID] = Object.keys(person);
+const peopleJSONAdapter = new PeopleJSONAdapter(person);
+const stories = storiesOf('Webex Avatar', module);
+
+// Stories
+stories.add('With no person status', () => <WebexAvatar personID={personID} adapter={peopleJSONAdapter} />);
+
+stories.add(`With '${PersonStatus.ACTIVE}' person status`, () => {
```

Comment:
I think the story names would be easier to read if it was just the status name.
```suggestion
stories.add(`${PersonStatus.ACTIVE}`, () => {
```